src/azure.js

```diff
@@ -44,14 +44,12 @@ builder.declare({
   let account = req.params.account;
   let continuationToken  = req.query.continuationToken || null;
 
-  await req.authorize({account});
-
```

Comment:
This is no longer necessary with scopeExpressions, right?

Reply:
Yeah. It should be fine. We'll get alerts if I'm wrong as well!